README.md

```diff
@@ -0,0 +1,33 @@
+# Examining Batch Effects on Cell Painting Data
+
+This project examines the presence of batch effects in (high-throughput) single-cell morphology data of different strains of cells subjected to various perturbations.
```

Comment:
```suggestion
This project examines the presence of technical artifacts in (high-throughput) single-cell morphology data of different cell lines subjected to various perturbations.
Technical artifacts include any nuisance signal in input data not explained by true biology.
```

Changing from "batch effects" to "technical artifacts" - do you agree that this is more descriptive?